vnext/Desktop.IntegrationTests/DesktopTestRunner.cpp

```diff
@@ -90,7 +90,7 @@ shared_ptr<ITestInstance> TestRunner::GetInstance(
   devSettings->platformName = "windows";
 
   // Set to JSIEngineOverride::Chakra when testing the Chakra.dll JSI runtime.
```

Comment:
>Chakra.dll JSI runtime. [](http://example.com/codeflow?start=55&length=23)

Should we switch it to V8? Or abstract it out to make Chakra-specific test?

Reply:
Sounds good. If it doesn't introduce test failures, I'll switch to V8.

Reply:
Actually we should probably run it for all supported engine overrides....